graphql/database/users/logEmailVerified.js

```diff
@@ -29,6 +34,21 @@ const logEmailVerified = async (userContext, userId) => {
   if (userContext.emailVerified) {
     try {
       await rewardReferringUser(userContext, userId)
+      const override = getPermissionsOverride(ADMIN_MANAGEMENT)
+      const originalInvitedUsers = await InvitedUsersModel.query(
+        override,
+        userContext.email
+      )
+        .usingIndex('InvitesByInvitedEmail')
+        .execute()
+      await Promise.all(
+        originalInvitedUsers.map(invitedUserEntry =>
+          InvitedUsersModel.update(override, {
+            ...invitedUserEntry,
+            invitedId: userId,
```

Comment:
if a person has been invited multiple times over a few months, do we invite all or just the latest?

Reply:
I think most useful to just update the one that was successful.